README.md

```diff
@@ -23,6 +23,13 @@ git clone https://github.com/TabbyML/quick-question.git
 # Save your OPENAI_API_KEY to file as secret
 echo YOUR_OPENAI_API_KEY > openai_api_key.txt
 
+# Setup your own codebase (Optional).
```

Comment:
`# Run tabby on your github project`

Reply:
Should be "Quick Question"?  
‘Run `tabby` on your github project’ -> 'Run `Quick Question` on your github project'